src/common/arrays_input.c

```diff
@@ -114,16 +132,26 @@ pgr_get_bigIntArr(ArrayType *v, size_t *arrlen, bool allow_empty) {
 
     pfree(elements);
     pfree(nulls);
+    PGR_DBG("Array size %ld", (*arrlen));
```

Comment:
This line was not there in the develop branch but present in the master branch. Is it's a mistake or that we don't require this debug msg now?

Reply:
Normally PGR_DBG is when somee problem is happening and we need to debug.
When the project is compiled without the -DDEBUG=ON (aka -DDEBUG=OFF) then the lines are ignored.